public/app/whiteboards/edit/edit.html

```diff
@@ -36,7 +36,8 @@ <h3 class="modal-title">Edit Whiteboard Settings</h4>
         </div>
         <div class="modal-footer">
           <button class="btn btn-default" data-ng-click="closeModal()">Cancel</button>
-          <button class="btn btn-primary" type="submit">Save Settings</button>
+          <button class="btn btn-primary" type="submit">Save settings</button>
+          <button class="btn btn-default" data-ng-click="deleteWhiteboard()">Delete whiteboard</button>
```

Comment:
@paulkerschen nit-picky UI question: I'd expect 'Save' on far right of this button set because it's the move-forward action.